telegram/ext/_jobqueue.py

```diff
@@ -41,6 +41,9 @@
     from telegram.ext import Application
 
 
+_day_tuple = tuple(range(7))
```

Comment:
```suggestion
_ALL_DAYS = tuple(range(7))
```

Reply:
I think it didn't allow me to make it capital for some reason